### ISSUE TITLE
Case fans plugged into the motherboard are not being detected. 

### ISSUE THREAD

**Issue by SirTwsted** [Rem0o/FanControl.Releases#2423]
**Describe the bug**
---------------------
I have a Gigabyte X670E Aorus extreme motherboard,  and I have 2 CPU fans on my deepcool air CPU cooler for the CPU.
I have 1 exhaust fan and I also have 6 case fans. 3 on top and 3 in the front of the case.
Fan Control ONLY see's 1 fan (I don't know which one that is) and the NVidia card fan.
I don't know why Fan Control doesn't see all my other fans.  They are all 120mm

**Is there a log.txt file next to FanControl.exe with recent date entries?**
--------------------------------------------------------------------------
There is no log.txt file in the folder.

**Relevant hardware specs and setup**
---------------------------------------
DeepCool Assassin 4 dual Fan
Gigabyte X670E Aorus Extreme motherboard  (Where the 7 case fans are all plugged into).
The CPU dual 120mm fans are plugged into the normal fan header for CPU Fans.
GForce GTX-1070-Ti - (yaya, I know its an old outdated card, I am saving up & waiting for the price to drop for the 7900XTX.)

Screenshot:
------------
![Screenshot_119](https://github.com/Rem0o/FanControl.Releases/assets/15247713/eb6727ca-65ea-4984-bc8a-6a73a08303c6)

Thanks for reading, its greatly appreciated.

**Comment by Rem0o** [comment]
How/where exactly are your fans plugged into your motherboard? Be specific.  What headers, any splitters/hub?

**Comment by SirTwsted** [comment]
They are all individually plugged into the headers on the motherboard directly.
The Gigabyte X670E Aorus Extreme has a lot of fan headers.
The motherboard is equip with 10*PWM/DC Fan Headers

**Comment by Rem0o** [comment]
Do you got this enabled? 
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/ea3ba550-7f45-4ef1-85e1-76b8fb436ed2)

**Comment by SirTwsted** [comment]
No, I did not have this enabled.  I clicked the checkmark box and still no changes.  🤷‍♀️

**Comment by Rem0o** [comment]
Do you have HWInfo? Open it up and see if the fan channels are on multiple chips, and which chips.

**Comment by SirTwsted** [comment]
I don't know if this is what you are looking for.
But its the only thing I can find in HWInfo regarding what you are asking.
-----------------------------------------------
![Screenshot_129](https://github.com/Rem0o/FanControl.Releases/assets/15247713/b449f7de-cb3e-47a0-9b8b-865e3d5bcb3c)

**Comment by LetMeDecay** [comment]
> ## I don't know if this is what you are looking for.
> But its the only thing I can find in HWInfo regarding what you are asking.
> ![Screenshot_129](https://private-user-images.githubusercontent.com/15247713/318581694-b449f7de-cb3e-47a0-9b8b-865e3d5bcb3c.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIwMzQ3NzYsIm5iZiI6MTcxMjAzNDQ3NiwicGF0aCI6Ii8xNTI0NzcxMy8zMTg1ODE2OTQtYjQ0OWY3ZGUtY2IzZS00N2EwLTliOGItODY1ZTNkNWJjYjNjLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDAyVDA1MDc1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQwZDA5ZjMzODRlMGI1OTA2MTMyOGUyNDY5NjliNjllZjBjZDAwMmNkYWU5MTUzMTYzOTZjZDVkZTI5ODVjMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5VnRXhoYm10NwPCihFAHcBoUphoJDh2GIk1MssXjaFg)

looks like hwinfo can't get info from io chip on your motherboard 
have you tried update bios to latest version and toggle all fans to pwm mode?
also update hwinfo to latest version

i have a werid problem on my b650m aorus pro board. 
LibreHardwareMonitor was unable to obtain rpm info after I installed the computer for the first time until I replugged and unplugged the fan

download LibreHardwareMonitor, you should get some info from io chip like this 
[LibreHardwareMonitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/852#issuecomment-1618626006)

**Comment by SirTwsted** [comment]
I have the latest bios on my motherboard.  And HWInfo is on the latest version.

**Comment by Amoeba00** [comment]
Do you have BIOS configured to manage the fans or are they set to default/auto?  Do you know if the fans are DC or PWM?  If DC, then you may need to manually set in BIOS to DC mode.

**Comment by SirTwsted** [comment]
No BIOS configuration, they are all default.  And as far as I can remember, they are all PWM fans,
